package.json

```diff
@@ -4,8 +4,8 @@
   "scripts": {
     "start": ". ./scripts/populate-git-env.sh && next dev --port 8888",
     "start:tunnel": "ngrok http 8888",
-    "build": "yarn test:once:group:no-integration && next build",
-    "build:profiler": ". ./scripts/populate-git-env.sh && next build --profile --debug",
+    "build": ". ./scripts/populate-git-env.sh && yarn test:once:group:no-integration && next build",
+    "build:profiler": "next build --profile --debug",
```

Comment:
This command shouldn't have been changed.

Reply:
Why not?

Reply:
That's true, a build:local is what we need.